AzureCalling/Controllers/JoinCallViewController.swift

```diff
@@ -10,29 +10,23 @@ class JoinCallViewController: UIViewController, UITextFieldDelegate {
     // MARK: Constants
 
     private let groupIdPlaceHolder: String = "ex. 4fe34380-81e5-11eb-a16e-6161a3176f61"
-    private let teamsLinkPlaceHolder: String = "ex. https://teams.microsoft.com/..."
 
     // MARK: Properties
 
     var callingContext: CallingContext!
 
-    private var joinCallType: JoinCallType = .groupCall
-
     // MARK: IBOutlets
 
     @IBOutlet weak var joinCallButton: UIRoundedButton!
-    @IBOutlet weak var joinIdTextField: UITextField!
-    @IBOutlet weak var contentStackView: UIStackView!
-    @IBOutlet weak var groupIdButton: UIButton!
-    @IBOutlet weak var meetingLinkButton: UIButton!
+    @IBOutlet weak var groupInputTextField: UITextField!
```

Comment:
We can keep the name as `joinIdTextField`. There will be less conflicts when we merge later on.